aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/MutationProcessor.java

```diff
@@ -41,15 +41,14 @@
  * {@link AppSync}. The responses to these mutations are themselves forwarded to the Merger,
  * which may again modify the store.
  */
-@SuppressWarnings("CodeBlock2Expr")
 final class MutationProcessor {
     private static final Logger LOG = Amplify.Logging.forNamespace("amplify:aws-datastore");
 
     private final VersionRepository versionRepository;
     private final Merger merger;
     private final AppSync appSync;
     private final MutationOutbox mutationOutbox;
-    private final CompositeDisposable disposable;
+    private final CompositeDisposable ongoingOperationsDispoable;
```

Comment:
```suggestion
    private final CompositeDisposable ongoingOperationsDisposable;
```

Reply:
Lol, when I was in my twenties, and the older engineers would make typos all over the place, I'd look at it, and be like "what is _up_ with these guys?" I understand it, now. It's a combination of tired eyes, tired hands, and too much to write. haha

Anyway, thanks! Will update.